news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md

```diff
@@ -0,0 +1,131 @@
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Swahili will be available in PrestaShop 1.7.8 :tada:
+
+I have the pleasure to announce to you that PrestaShop will finally be available in Swahili in the 1.7.8 version! I thus encourage you to translate, suggest and proofread the PrestaShop Swahili 1.7.7 translation project, as that will make it available to you and worldwide users as well. Currently, there aren’t many Swahili translators on PrestaShop’s Crowdin community, so if you know any Swahili natives, spread the word and do not hesitate to invite them to participate in the [PrestaShop Swahili translations project](https://crowdin.com/project/prestashop-official/sw#), they will be more than welcome! :heart_eyes:
+
+### Daily language packs updates
+
+Many of you asked why the translated strings were not updated in their shop, even though they have translated and approved strings, so I would like to take this opportunity to explain to you how translation packs are updated in PrestaShop now.
+
+Translations are updated everyday with a 24 hours delay. Every morning our developers send an export request to Crowdin, but since the exporting process takes some time, they download the export made the day before and send the files to [Github](https://github.com/PrestaShop/TranslationFiles). When the repository is updated, a robot will gather the translation files and make them available to download. 
+But one detail to take into account is that only translations of the last version, the 1.7.7, are updated. 
+
+Since PrestaShop 1.7.7 hasn’t been officially released yet, what you can do is to translate and approve strings of the 1.7.7 version, retrieve these strings from translationfiles and upload them on: app/Resources/Translations/{localeCode}. 
+It might be inconvenient for now but PrestaShop's developers team have in the backlog projects to make all new translations available for all shops from the 1.7.5 version and on, so let’s keep a positive mind! :sparkles:
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, the PrestaShop team would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: choefer, Çocuk Saatleri (cocuksaatleri), nenowasaleh, Pacho Gonzalez (Pachogzz), Puunaama1 and Stella Georgakopoulou (Stella_Geo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+
+
+## A few stats
+
+* **37 members** were active on the project this month.
```

Comment:
And 38 active members.

Reply:
So 37 members without Nicolas Etienne :)